__tests__/app.test.js

```diff
@@ -68,3 +68,64 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("PATCH: /api/articles/:article_id", () => {
+  test("200 should increase the votes of the article by the incremented amount", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 1 })
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article.votes).toBe(101);
+      });
+  });
+  test("200 should decrease the votes of the article by the decremented amount", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: -5 })
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article.votes).toBe(95);
+      });
+  });
+  test("200 should display the new and patched article", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: -5 })
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article.title).toEqual("Living in the shadow of a great man");
+        expect(article.topic).toEqual("mitch");
+        expect(article.author).toEqual("butter_bridge");
+        expect(article.body).toEqual("I find this existence challenging");
+        expect(article.created_at).toEqual("2020-07-09T20:11:00.000Z");
+        expect(article.votes).toEqual(95);
+        expect(article.article_img_url).toEqual(
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+        );
+      });
+  });
+  test("404 returns an valid article_id which does not exist", () => {
+    return request(app)
+      .patch("/api/articles/100")
+      .send({ inc_votes: 50 })
+      .expect(404)
+      .then(({ body }) => {
+        const { message } = body;
+        expect(message).toBe("Article Id not found");
+      });
+});
```

Comment:
Could we pop another 400 test in here please? For if we try and patch an invalid ID, e.g. articles/banana 👍